include/psa/crypto_struct.h

```diff
@@ -239,18 +239,17 @@ typedef struct {
     psa_key_type_t MBEDTLS_PRIVATE(type);
     psa_key_bits_t MBEDTLS_PRIVATE(bits);
     psa_key_lifetime_t MBEDTLS_PRIVATE(lifetime);
-    mbedtls_svc_key_id_t MBEDTLS_PRIVATE(id);
     psa_key_policy_t MBEDTLS_PRIVATE(policy);
     psa_key_attributes_flag_t MBEDTLS_PRIVATE(flags);
+    mbedtls_svc_key_id_t MBEDTLS_PRIVATE(id);
```

Comment:
Ok, but please add a comment, both here and in `struct psa_key_attributes_s`, explaining why that particular field is last. Otherwise there's a risk that someone will innocently move the fields again, e.g. if it improves the code size.

Reply:
Comments added.